src/pages/docs/v3/migration-guide.mdx

```diff
@@ -0,0 +1,236 @@
+---
+navigationLabel: Migration Guide
+order: 8
+---
+
+# Migrating from Popper 2 to Popper 3
+
+This page will help you migrate from Popper 2 to Popper 3.
+
+## Motivation
+
+The new API of Popper 3 was rebuilt from the ground up to be:
+
+- Platform-agnostic (can run on the web and native)
+- Lower-level and more powerful
+- Less magical, features are added as you need them
+- Much smaller in size, and tree-shakeable by default
+
+## 1. Change your dependencies
+
+Popper 3 makes the library platform-agnostic, which means it can run on native
+applications that translate JavaScript, like React Native. You now install
+`@popperjs/dom` to use it on the web, which has `@popperjs/core` as a
+dependency:
+
+```shell
+# With npm
+npm uninstall @popperjs/core
+npm i @popperjs/dom
+
+# With Yarn
+yarn remove @popperjs/core
+yarn add @popperjs/dom
+```
+
+## 2. Change imports
+
+Popper 2:
+
+```js
+import { createPopper } from '@popperjs/core';
+```
+
+Popper 3:
+
+```js
+import { position } from '@popperjs/dom';
+```
+
+## 3. Styling is up to you
+
+Popper 3 utilizes an inversion-of-control API, which means it is now pure and
+leaves the application of styles up to you. The "magic" of Popper 2 made it
+harder to debug, and setting opinionated defaults can be annoying to adjust.
+
+Popper 2:
+
+```js
+createPopper(reference, popper);
+```
+
+Popper 3:
+
+```js
+function applyStyles({ x = 0, y = 0, strategy = 'absolute' }) {
```

Comment:
I think `position` here would make everything easier.

Reply:
You mean the ordering of the code?

Reply:
`position` instead of `strategy`, this way we only return proper CSS properties from the function. And people don't have to manually map strategy to position

Reply:
Since it's destructuring the property, we'd still need to map it in the variable like `strategy: position = 'absolute'`, which we could do

Reply:
What I'm proposing is

```
function applyStyles({ x = 0, y = 0, position = 'absolute' }) {

}
```

Reply:
That means `position()` would return `{ position: strategy }`, so it's mapping it for us? I was not sure if that was expected or not